### ISSUE TITLE
Possibility to remove product from contract when in proposed phase

### ISSUE THREAD

**Issue by MaikoCheng** [GEWIS/parelpracht-client#63]
Correct me if I'm wrong but I would still like to remove the products from a contract when the contract is in "proposed" phase. I put the contracts in this phase when I have proposed these products to the company. When the company then e.g. does not want the Direct mailing anymore (see C1272), I have to cancel the product, and then the financial amounts still stay the same and the products on the contract still show on the definite contract I'd make.

tldr: Please let me remove products when the contract is in proposed phase.

**Comment by Yoronex** [comment]
Implemented in https://github.com/GEWIS/parelpracht-client/pull/74.
